src/libmongoc/tests/unified/runner.c

```diff
@@ -26,6 +26,86 @@
 #include "util.h"
 
 
+typedef struct {
+   const char *file_description;
+   const char *test_description;
+} skipped_unified_test_t;
+
+#define SKIP_ALL_TESTS NULL
+
+/* clang-format off */
+skipped_unified_test_t SKIPPED_TESTS[] = {
+   /* CDRIVER-3630: libmongoc does not unconditionally raise an error when using
```

Comment:
Ah right, IIRC we only raise an error if the server version indicates that hint is not supported.